src/pages/components/inline-loading/accessibility.mdx

```diff
@@ -14,197 +14,66 @@ import {
   ListItem,
 } from '@carbon/react';
 
+<PageDescription>
+
+No accessibility annotations are needed for inline loading components, but keep these considerations in mind if you are modifying Carbon or creating a custom component.
+
+</PageDescription>
+
 <AnchorLinks>
-  <AnchorLink>How it works</AnchorLink>
-  <AnchorLink>Accessibility considerations</AnchorLink>
-  <AnchorLink>Resources</AnchorLink>
-  <AnchorLink>Accessibility testing</AnchorLink>
+
+<AnchorLink>What Carbon provides</AnchorLink>
+<AnchorLink>Development considerations</AnchorLink>
+
 </AnchorLinks>
 
-## How it works
+## What Carbon provides
 
-The Carbon inline loading component is used to notify users that their action is
-being processed. W3C WAI-ARIA `role="alert"` and `aria-live="assertive"` are
-used to ensure screen reader users are also provided the same information.
+Carbon bakes keyboard operation into its components, improving the experience of
+blind users and others who operate via keyboard. Carbon also incorporates other
+accessibility considerations, some of which are described below.
 
-## Accessibility considerations
+### Keyboard interaction
 
-This component has been validated to meet the
-[WCAG 2.0 AA](https://www.w3.org/TR/WCAG20/) and
-[Section 508](http://www.section508.gov/) accessibility guidelines, however
-changes made by the content owner can affect accessibility compliance. Be sure
-to review and follow the guidance in this section when updating or adding new
-content to this component.
+The inline loading component is not itself keyboard operable; however, when inline loading is in an active state, it may temporarily replace or disable an associated component. For instance, if inline loading is showing the result of a button action, relevant input controls as well as other actions may be disabled (and so be inoperable) until the loading state changes to either error or complete.
```

Comment:
The button switching to disabled when inline loading is showing isn't baked into the component. We have a story showing how a user could code that up themselves but it isn't a feature of the component itself.

 I'm not sure how to update this guidance to reflect that. I just worry that people will think that it is inline loading that is disabling relevant input controls when that would need to be implemented separately by users

Reply:
Maybe something like, "We recommend disabling relevant input controls when inline loading is showing due to the result of a button action. To see an example of how to do this, you can check out our storybook [here](https://carbon-components-react.netlify.app/?path=/story/components-inlineloading--ux-example)"?

Reply:
Abbey, I assume you're talking about this:
> The inline loading component is not itself keyboard operable; however, when inline loading is in an active state, it may temporarily replace or disable an associated component. For instance, if inline loading is showing the result of a button action, relevant input controls as well as other actions may be disabled (and so be inoperable) until the loading state changes to either error or complete.

I think the use of "may" here conveys what's going on. Remember, we're just trying to capture the effects on keyboard operation -- the gist of this being that if something is disabled, obviously it can't be keyboard accessible. I'll review the language and see if it can't be made clearer that this disabling function is NOT part of Carbon.

Reply:
How about "... however, when inline loading is in an active state, it can be implemented to it temporarily..."

> The inline loading component is not itself keyboard operable; however, when inline loading is in an active state, it can be implemented so that it temporarily replaces or disables an associated component. For instance, if inline loading is showing the result of a button action, relevant input controls as well as other actions may be disabled (and so be inoperable) until the loading state changes to either error or complete.